module/applications/_module.mjs

```diff
@@ -13,6 +13,7 @@ export * as shared from "./shared/_module.mjs";
 
 export {default as Accordion} from "./accordion.mjs";
 export {default as Award} from "./award.mjs";
+export {default as BastionConfig} from "./bastion.mjs";
```

Comment:
Perhaps we should add a `settings` folder to hold these settings apps (and `CompendiumBrowserSourceConfig`.

Reply:
Good idea. Will do it later though